.github/workflows/unit-tests.yml

```diff
@@ -8,8 +8,8 @@ on:
 
 env:
   BUILD_TYPE: Debug
-  THUNDER_REF: "R2-v1.14"
-  INTERFACES_REF: "R2-v1.14"
+  THUNDER_REF: "39c82df24c7b4a4a10663cf527a3b289e7c1ceae"
```

Comment:
Checking why do we need this version. R2-v1.14 is later than 39c82df24c7b4a4a10663cf527a3b289e7c1ceae. Pls check https://github.com/rdkcentral/Thunder/commits/R2

Reply:
@anand-ky 
changed THUNDER_REF and INTERFACES_REF with the default version please review it